src/clientcommands.cpp

```diff
@@ -106,6 +108,63 @@ void ClientCommands::sync()
   download( lp.projectNamespace, lp.projectName );
 }
 
+static void printVal( bool isJsonFormat,
+                      QTextStream &out,
+                      const QString &key,
+                      const QString &val,
+                      bool isLast )
+{
+
+  QChar separator;
+  QChar quote;
+  QString space;
+  if ( isJsonFormat )
+  {
+    separator = ',';
+    quote = '"';
+  }
+  else
+  {
+    separator = '\n';
+    space = " ";
+  }
+
+  out << quote << key << quote << ":"  << space << quote << val << quote
+      ;
+
+  if ( !isLast )
+    out << separator;
+}
+
+void ClientCommands::info( bool isJsonFormat )
+{
+  Q_ASSERT( isAuthorized() );
+  LocalProject lp = mLocalProjectsManager.projectFromDirectory( QDir::currentPath() );
+  if ( !lp.isValid() )
+    throw QString( "no mergin project in the current directory" );
+
+  QTextStream out( stdout );
+
+  QChar prefix;
+  QChar suffix;
+  if ( isJsonFormat )
+  {
+    suffix = '}';
+    prefix = '{';
+  }
+  else
+  {
+    suffix = '\n';
+  }
+
+  out << prefix;
+  printVal( isJsonFormat, out, "name", lp.projectName, false );
+  printVal( isJsonFormat, out, "namespace", lp.projectNamespace, false );
+  printVal( isJsonFormat, out, "id", lp.id(), false );
+  printVal( isJsonFormat, out, "localVersion", QString::number( lp.localVersion ), true );
```

Comment:
Can we do the JSON output properly, through QJsonDocument + other Qt classes? Like this, we would have encoding problems if a value contains double qoutes, we don't support arrays etc.

Not having a single `printVal` function for both JSON and human-oriented output would give us more flexibility how we format output for the user...